README.md

```diff
@@ -1 +1,54 @@
-# openstack-test-operator
\ No newline at end of file
+openstack-tempest-operator
```

Comment:
i’m convinced that “opensatck-tempest-operator” is the best choice for the name, won’t it get confused with test-operator?

"openstack-tempest-operator" will be confused with test-operator, tempest and is not technically correct because we have also another test runner - tobiko (not tempest, not tempest plugin either) .. over time, test-operator may include more test runners beside tempest and tobiko

Reply:
as discussed on slack, “operator” might not be a good choice either as this won’t be actual operator , based on the names someone might think that it belongs to the same bucket as nova-operator, <service>-operator, test-operator but it doesn’t because this is not an operator

what about openstack-skiplist-manager? Or anything similar, let’s just avoid “tempest” , “test” and “operator” and rather incorporate “skiplist” into the name (as we were used to that in exactly this context)

Reply:
https://redhat-internal.slack.com/archives/C03MD4LG22Z/p1719388392481519

Following this discussion -openstack-tests-list sounds good